pkgs/by-name/go/godini/package.nix

```diff
@@ -0,0 +1,49 @@
+{
+  buildGoModule,
+  fetchFromGitHub,
+  godini,
+  lib,
+  nix-update-script,
+  testers,
+}:
+
+buildGoModule rec {
+  pname = "godini";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "bilbilak";
+    repo = "${pname}";
+    rev = "v${version}";
```

Comment:
```suggestion
    tag = "v${version}";
```

Reply:
I copied that straight out of the [documentation](https://nixos.org/manual/nixpkgs/stable/#sec-language-go) and it worked too! But I understand that `tag` is more accurate and it passed my test as well.

Reply:
Yeah, Nixpkgs's documentation is a bit dated... We really need to get on updating them as a part of sweeping treewide changes